docs/hyproof/api-flow.md

```diff
@@ -0,0 +1,279 @@
+## HyProof Api Flow
+
+This document describes the API flow that can be used to demonstrate the usage of the HyProof project.
+
+---
+
+## HyProof Api Flow: Prerequisites
+
+In order to go trough the entire flow, there are a couple of prerequisites required, described below.
+
+### HyProof Api Flow: Prerequisites: Services
+
+In order to be able to reproduce the steps described in this document you need to have the three personas setup ( a complex setup that include all the needed services for three personas ). For that, please refer to the **[docker-compose-3-personal.yml](https://github.com/digicatapult/dscp-hyproof-api/blob/main/docker-compose-3-personal.yml)** file found in the **[dscp-hyproof-api](https://github.com/digicatapult/dscp-hyproof-api)** repository.
+
+To run this, a command similar to the next one can be used:
+
+```sh
+docker-compose -f docker-compose-3-personal.yml down -v && docker-compose -f docker-compose-3-personal.yml up -d && \
+npm i && npm run db:migrate && \
+npm run flows
```

Comment:
This is incorrect. There's no need to run the migration or flows scripts in the 3 persona docker as these are run as part of brining up services